docs/src/index.tsx

```diff
@@ -15,46 +15,12 @@ import Router from './routes'
 const mountNode = document.createElement('div')
 document.body.appendChild(mountNode)
 
-const globalRules = siteVars => ({
-  html: {
-    fontSize: siteVars.htmlFontSize,
-  },
-  body: {
-    padding: siteVars.bodyPadding,
-    margin: siteVars.bodyMargin,
-    fontFamily: siteVars.bodyFontFamily,
-    fontSize: siteVars.bodyFontSize,
-    lineHeight: siteVars.lineHeightBase,
-  },
-})
-
-// TODO: formalize fonts into siteVariables/theme contract.  Leaving hard coded for now.
-const fontFaces = [
-  {
-    name: 'Segoe UI',
-    paths: ['public/fonts/segoe-ui-regular.woff2'],
-    style: { fontWeight: 400 },
-  },
-  {
-    name: 'Segoe UI',
-    paths: ['public/fonts/segoe-ui-semibold.woff2'],
-    style: { fontWeight: 600 },
-  },
-  {
-    name: 'Segoe UI',
-    paths: ['public/fonts/segoe-ui-bold.woff2'],
-    style: { fontWeight: 700 },
-  },
-]
+console.log('THEME', theme)
```

Comment:
seems to be a debug leftover

Reply:
Thanks, branch was still in progress until right now 👍